tutorials/setup.md

```diff
@@ -188,6 +189,20 @@ make BUILDTAGS=""
 sudo make install
 ```
 
+### Install with Ansible
+
+An [Ansible Role](https://github.com/alvistack/ansible-role-cri_o) is also available to automate the above steps:
```

Comment:
Do you think it would make sense to add the role to this repository here (as well)?

Reply:
To be honest the role will not works in solo: it also depends on other roles, e.g. golang, git, runc, conmon, etc (see https://github.com/alvistack/ansible-role-cri_o/blob/master/ansible-role-requirements.yml).

Why the steps look such simple now, just because molecule will fetch the dependency before the converge start. So the truth is you will have more than single Ansible Role being used. IMHO, in this repo we may not hope to locally embed too many Ansible Role dependency, it shouldn’t be this repo target. 

Moreover for production deployment use cases, you may hope to use my Ansible Collection for Kubernetes (see https://github.com/alvistack/ansible-collection-kubernetes/blob/master/playbooks/cri_o-install.yml), for multi target DevOps management. BTW, promote the Ansible Collection here may again overkill.

Reply:
Alright thanks for the clarification. Sounds good to me